fighters/pitb/src/acmd/aerials.rs

```diff
@@ -1,6 +1,59 @@
 
 use super::*;
 
+#[acmd_script( agent = "pitb", script = "effect_attackairn", category = ACMD_EFFECT, low_priority )]
+unsafe fn pitb_attack_air_n_effect(fighter: &mut L2CAgentBase) {
+    let lua_state = fighter.lua_state_agent;
+    let boma = fighter.boma();
+    frame(lua_state, 4.0);
+    if is_excute(fighter) {
+        EFFECT_FOLLOW(fighter, Hash40::new("pitb_atk_air_n"), Hash40::new("bowr"), 0, 0, 0, -90, 90, 0, 0.9, true);
+        EFFECT_FOLLOW(fighter, Hash40::new("pitb_sword"), Hash40::new("bowr"), 0, 0.87, 0, 0, 90, 0, 1, true);
+        EFFECT_FOLLOW(fighter, Hash40::new("pitb_sword"), Hash40::new("bowr"), 0, -0.87, 0, 180, -90, 0, 0.9, true);
+    }
+    frame(lua_state, 10.0);
+    if is_excute(fighter) {
+        EFFECT_OFF_KIND(fighter, Hash40::new("pitb_atk_air_n"), true, true);
+    }
+    frame(lua_state, 23.0);
+    if is_excute(fighter) {
+        EFFECT_FOLLOW(fighter, Hash40::new("pitb_atk_air_n"), Hash40::new("bowr"), 0, 0, 0, -90, 90, 0, 1.1, true);
+        AFTER_IMAGE4_ON_arg29(fighter, Hash40::new("tex_pitb_sword1"), Hash40::new("tex_pitb_sword2"), 3, Hash40::new("swordr1"), 0.0, 1.0, -0.2, Hash40::new("swordr1"), 0.0, 11.4, -1.2, true, Hash40::new("null"), Hash40::new("swordr1"), 0.0, 0.0, 0.0, 0.0, 90.0, 0.0, 1.0, 0, *EFFECT_AXIS_X, 0, *TRAIL_BLEND_ALPHA, 101, *TRAIL_CULL_NONE, 1.4, 0.1);
+    }
+    frame(lua_state, 24.0);
+    if is_excute(fighter) {
+        EFFECT_OFF_KIND(fighter, Hash40::new("pitb_sword"), false, false);
+        EFFECT(fighter, Hash40::new("sys_flash"), Hash40::new("top"), 0, 7, 8, 0, 0, 0, 0.9, 0, 0, 0, 0, 0, 0, true);
+        LAST_EFFECT_SET_RATE(fighter, 1.5);
+    }
+    frame(lua_state, 25.0);
+    if is_excute(fighter) {
+        EFFECT_OFF_KIND(fighter, Hash40::new("pitb_atk_air_n"), true, true);
+    }
+    frame(lua_state, 27.0);
+    if is_excute(fighter) {
+        AFTER_IMAGE_OFF(fighter, 2);
+    }
+}
+
+#[acmd_script( agent = "pitb", script = "sound_attackairn", category = ACMD_SOUND, low_priority )]
+unsafe fn pitb_attack_air_n_sound(fighter: &mut L2CAgentBase) {
+    let lua_state = fighter.lua_state_agent;
+    let boma = fighter.boma();
+    frame(lua_state, 4.0);
+    if is_excute(fighter) {
+        PLAY_STATUS(fighter, Hash40::new("se_pitb_attackair_n01"));
+    }
+    wait(lua_state, 14.0);
```

Comment:
please use `frame` instead of `wait` here. You are targeting a specific frame with this call (smell: wait is longer that a couple frames)